Sources/TSCTestSupport/SwiftTestingHelpers.swift

```diff
@@ -0,0 +1,27 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Testing
```

Comment:
This will fail on CI, which uses Swift 5.8 and doesn't have `swift-testing` included.

Reply:
Are the pipelines for the project using Swift 6.0 yet?